content/_includes/ClusterSetup.md

```diff
@@ -11,12 +11,11 @@ Follow this procedure for each TrueNAS SCALE system that is to be connected to T
    If not, click **Create Pool** and make a new pool using any of the available disks.
 
 2. Go to the **Network** page and look at the **Interfaces** card.
-   
-   a. Ensure two interfaces are available and note which is the primary interface that allows SCALE web interface access and access between SCALE systems, TrueCommand, and Active Directory environments.
-      Having two interfaces allows connecting the SCALE systems to Active Directory and using TrueCommand to create and manage the cluster.
 
-   b. Ensure the second interface has a static IP address on a different network/subnet that connects all the SCALE systems.
-      This interface securely handles all the data-sharing traffic between the clustered systems.
+   <ol type="a">
+   <li>Ensure two interfaces are available and note which is the primary interface that allows SCALE web interface access and access between SCALE systems, TrueCommand, and Active Directory environments. Having two interfaces allows connecting the SCALE systems to Active Directory and using TrueCommand to create and manage the cluster.</li>
```

Comment:
These do read like action steps, but if they don't have to be completed in a particular sequence you can use bullet lists for this content, as long as it is indented under step 2. If this is an action they need to complete in a specific sequence, make them a nested number step. Also I'm not fond of Ensure as the action because they might not know how to "ensure". Do we need to tell them how to do this? Just asking for a friend. ;-)~